src/data/java/net/dries007/tfc/data/providers/BuiltinRecipes.java

```diff
@@ -271,6 +271,12 @@ public void accept(ResourceLocation id, Recipe<?> recipe, @Nullable AdvancementH
             add(new LandslideRecipe(BlockIngredient.of(
                 blocks.get(SoilBlockType.MUD).get()
             ), blocks.get(SoilBlockType.MUD).get().defaultBlockState()));
+            add(new LandslideRecipe(BlockIngredient.of(
+                blocks.get(SoilBlockType.CRACKED_EARTH).get()
+            ), blocks.get(SoilBlockType.CRACKED_EARTH).get().defaultBlockState()));
+            add(new LandslideRecipe(BlockIngredient.of(
+                blocks.get(SoilBlockType.SALTED_EARTH).get()
+            ), blocks.get(SoilBlockType.SALTED_EARTH).get().defaultBlockState()));
```

Comment:
These, given they are 1-1 landslide recipes, shouldn't exist, and should use the `can_landslide` tag instead (which automatically infers copying the input state). It's just less files which is nice. Also the mud one above should be doing this too please and thanks.

Reply:
Neat, didn't know we had that but that's really nice